tests/e2e/sample-applications/virtual-machines/cirros-test/3-vms/cirros-datasource-datavolume.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: cdi.kubevirt.io/v1beta1
+  kind: DataSource
+  metadata:
+    name: cirros
+    namespace: openshift-virtualization-os-images
+  spec:
+    source:
+      pvc:
+        name: cirros
+        namespace: openshift-virtualization-os-images
+- apiVersion: cdi.kubevirt.io/v1beta1
+  kind: DataVolume
+  metadata:
+    name: cirros
+    namespace: openshift-virtualization-os-images
```

Comment:
suggest we use quay.io images  that are usually  available in all env.
quay.io/kubevirt/cirros-container-disk-demo